core/backend/eslint.config.js

```diff
@@ -0,0 +1,20 @@
+const iTwinPlugin = require("@itwin/eslint-plugin");
+
+module.exports = [
+  {
+    files: ["**/*.{ts,tsx}"],
```

Comment:
`*.tsx` extension is for typescript react components, which shouldnt exist at all in this repo, since the core repo should be just a ui framework agnostic and a pure js library